py/variable.cpp

```diff
@@ -20,18 +20,32 @@ static PyObject*
 Variable_new( PyTypeObject* type, PyObject* args, PyObject* kwargs )
 {
 	static const char *kwlist[] = { "name", "context", 0 };
-	PyObject* name;
 	PyObject* context = 0;
+
+#if PY_MAJOR_VERSION >= 3
+	const char* name;
```

Comment:
Do you mean `s` or `s#` is the correct one? I don't think we should support anything but unicode variable names on Py3. `s#` also allows null bytes mid-string, which may cause other problems down the road.